src/components/onboarding/screens/secret-key.tsx

```diff
@@ -32,7 +32,23 @@ export const SecretKey: React.FC<SecretKeyProps> = props => {
   });
 
   return (
-    <>
+    <form
```

Comment:
Thomas recently added an `onSubmit` prop to screen, see pages like `Username` for an example. It'll generate a `<form>` for you.